playbooks/common/openshift-master/config.yml

```diff
@@ -47,6 +47,7 @@
           public_hostname: "{{ openshift_public_hostname | default(None) }}"
           public_ip: "{{ openshift_public_ip | default(None) }}"
           deployment_type: "{{ openshift_deployment_type }}"
+          portal_net: "{{ openshift_portal_net | default(openshift_master_portal_net) | default(None) }}"
```

Comment:
Youl'll want to migrate the previously set fact similar to what I've done in https://github.com/openshift/openshift-ansible/pull/1550: https://github.com/detiber/openshift-ansible/blob/docker_refactor/roles/openshift_facts/library/openshift_facts.py#L31-L58